.gitignore

```diff
@@ -21,3 +21,6 @@ _testmain.go
 
 *.exe
 *.test
+
+# IDE files
+/.idea/
```

Comment:
Can we not include this in the PR? Project git ignore shouldn't be for ignoring your systems files made. There is a global git ignore for that. Also don't want to introduce different concepts in the PR.

Reply:
> global git ignore

Wow, first time I'm hearing about this